packages/manager/modules/pci/src/projects/project/public-gateways/onboarding/onboarding.constants.js

```diff
@@ -0,0 +1,20 @@
+export const GUIDES = [
+  {
+    id: 'creat-private-network-with-gateway',
+    link:
```

Comment:
You need to add support for all subs even though you have only one sub as of now. Please refer to Kubernetes onboarding page for code example.

Reply:
@radireddy I have taken care of this in config page as suggested by you.